activesupport/lib/active_support/encrypted_configuration.rb

```diff
@@ -22,6 +22,13 @@ def read
       ""
     end
 
+    # Write +contents+ with validation
+    def write(contents)
+      validate(contents)
```

Comment:
Why the need for the separate method? Shouldn't this just call `deserialize`?

Reply:
I thought that it would be easier to use if the method for validate was intentionally divided when creating a class that inherits `EncryptedConfiguration`.  But `deserialize` calls do enough. 
I'd like to add `content_path` to `YAML.load` of `deserialize` so that it can display the path. How do you think?

Reply:
Yeah, let's just add `content_path` to the load `deserialize` and call it a day. Much simpler patch then too :)

Reply:
Thanks for you comment! I updated!